config/default.yaml

```diff
@@ -21,7 +22,7 @@ config:
     #    credentials: username:password
         baseUrl: https://jenkins.example.com
         saveCache: true
-        projectFormat: build-{project:alpha}-snapshot.yml
+        projectFormat: '{project:alpha}.yml'
```

Comment:
:+1:

Reply:
Why do you need the single quotes?

Reply:
without them yaml thinks this is an object { "project": "alpha" }

Reply:
got it!